src/renderer/l10n/de.js

```diff
@@ -16,21 +16,18 @@ export default {
   addressBook: 'Adressbuch',
   all: 'Alle',
   amount: 'Menge',
-  amountCommitted: 'Amount Committed',
```

Comment:
We should collect all these lines being removed into another file to give to translators along with whichever others are missing that are in the English file.

Reply:
I've been thinking about that actually. I think we can write a script to do a diff on the keys and produce a new german file with english placeholders